build.gradle

```diff
@@ -166,6 +179,13 @@ dependencies {
 
 }
 
+//dependencyManagement {
+//    imports {
+//        mavenBom "org.keycloak.bom:keycloak-adapter-bom:${keycloakVersion}"
+//    }
+//}
+
+
```

Comment:
This section should be removed

Reply:
Ok, please see my last commits on this PR.